src/vmm/src/rpc_interface.rs

```diff
@@ -427,10 +442,32 @@ impl RuntimeApiController {
     }
 
     #[cfg(target_arch = "x86_64")]
-    fn create_snapshot(&mut self, params: CreateSnapshotParams) -> ActionResult {
+    fn create_snapshot(&mut self, create_params: CreateSnapshotParams) -> ActionResult {
         let mut locked_vmm = self.vmm.lock().unwrap();
-        persist::create_snapshot(&mut locked_vmm, params, VERSION_MAP.clone())
-            .map_err(VmmActionError::CreateSnapshot)
+        let create_start_us = utils::time::get_time(utils::time::ClockType::Monotonic) / 1000;
```

Comment:
nit: `utils::time::get_time(utils::time::ClockType::Monotonic) / 1000` shows up a lot, maybe it deserves a helper fn in `utils::time` - `get_time_us(ClockType)`?

Reply:
Done, also renamed `get_time` to `get_time_ns` to make things clearer.